WcaOnRails/app/jobs/sync_mailing_lists_job.rb

```diff
@@ -26,8 +26,8 @@ def perform
     GsuiteMailingLists.sync_group("results@worldcubeassociation.org", Team.wrt.current_members.includes(:user).map(&:user).map(&:email))
     GsuiteMailingLists.sync_group("software@worldcubeassociation.org", Team.wst.current_members.includes(:user).map(&:user).map(&:email))
     GsuiteMailingLists.sync_group("software-admin@worldcubeassociation.org", Team.wst_admin.current_members.includes(:user).map(&:user).map(&:email))
-    translators = User.where(id: TranslationsController::VERIFIED_TRANSLATORS_BY_LOCALE.values.flatten)
-    GsuiteMailingLists.sync_group("translators@worldcubeassociation.org", translators.map(&:email))
+    translator_users = UserGroup.translators.map(&:roles).flatten.map(&:user)
```

Comment:
I feel like any given user group (same applies to Delegates, Team Members, etc...) should have an easy convenience method for getting the users inside that group.
Why is the `flatten` even necessary? And if there is a good reason why it is necessary, why aren't you just using `flat_map(&:roles)` directly?

Reply:
Oh I didn't knew about `flat_map`. I changed it to `flat_map` now. I also created `users` method in `user_group`. The reason why flatten is there is because `UserGroup.translators` will give the list of user_groups which belongs to `group_type` translators. Currently there is only one group under this, but I thought still using `flatten` is better in long term.